### PR TITLE
Removes Nar-Sie from portal storm event

### PR DESCRIPTION
**Pull request by Robustin**
Who the fuck is ever going to run an event that kills everyone and immediately ends the round? If I want to spawn Nar-Sie ill just use the game panel. 

Conclusion: It takes me 5 seconds to summon a nar-sie, I don't need an event button to do that for me, but realistically no admin can pull off a portal storm without this event code and nobody would ever want to spawn Nar-Sie as an event. So, I'm trying to salvage this useless button from being something other than deadmin bait. 

Now called "Portal Storm: Constructs" to satisfy certain supernerds.

### CODE REVIEW

**Comment by MMMiracles** [comment]
someone actually thought that spawning nar-sie from a random event was a good idea

memes

**Comment by kilkun** [comment]
it's for deep lore admin events tbh. :-1:

**Comment by TehZombehz** [comment]
I could have sworn that was removed ages ago. I guess it was stealthily retained for memes.

**Comment by xxalpha** [comment]
This is not a random event, this is an admin event.

Close this PR.

File: code/modules/events/portal_storm.dm [hunk 1]
```diff
@@ -17,8 +17,8 @@
 	max_occurrences = 0
```
Comment (xxalpha) [inline code/modules/events/portal_storm.dm]: learn to read the code

**Comment by xxalpha** [comment]
If you don't like it create your own event. Stop being a faggot jesus christ.

**Comment by xxalpha** [comment]
It's called "Portal Storm: Nar-sie", if you're too stupid to read the buttons you are pressing just get deadmined.

**Comment by Robustin** [comment]
Ah the xxAlpha and Gunhog thumbsdown, the strongest guarantee that your PR is improving the game.

I knew it was an admin only event dipshit. There was a cult round where the cult got wiped out but I wanted to give the crew something ~thematic~ to fight near the end of the round. I considered this portal storm but LOL IT SPAWNS NAR SIE SO YOU CAN NEVER JUSTIFY TRIGGERING IT.

I'm trying to salvage this shit event so its not just another useless "DONT PRESS THIS" event cluttering up the menu. its either that or I can put it this code in a "removes shitty events that nobody would ever run" PR.

File: code/modules/events/portal_storm.dm [hunk 2]
```diff
@@ -11,14 +11,14 @@
...
 	typepath = /datum/round_event/portal_storm/portal_storm_narsie
 	weight = 0
 	max_occurrences = 0
 
 /datum/round_event/portal_storm/portal_storm_narsie
-	boss_types = list(/obj/singularity/narsie/large = 1)
-	hostile_types = list(/mob/living/simple_animal/hostile/construct/armored/hostile = 10,\
+	boss_types = list(/mob/living/simple_animal/hostile/construct/armored/hostile = 2)
+	hostile_types = list(/mob/living/simple_animal/hostile/construct/armored/hostile = 8,\
 						/mob/living/simple_animal/hostile/construct/wraith/hostile = 10)
```
Comment (ChangelingRain) [inline code/modules/events/portal_storm.dm]: You should add a few hostile artificers, like 3-4. They run from everything but will find and heal constructs.

Comment (Robustin) [inline code/modules/events/portal_storm.dm]: yea I forgot that their AI was actually pretty good

**Comment by Bawhoppen** [comment]
wow robustin I literally did not think you could get more retarded 'WHY ARE THERE BUTTONS I CANT PRESS REMOVE'

**Comment by Robustin** [comment]
Because I saw a utility in a portal storm event that summoned constructs but not Nar-Sie, is that so hard to understand? Apparently it is given how xxalpha was so overcome with rage at seeing his defunct code altered. 

THIS BUTTON HAS NO PURPOSE, I AM GIVING IT PURPOSE, WHY IS THIS A DIFFICULT CONCEPT FOR SOME OF YOU?

**Comment by Archie700** [comment]
So did people actually read the coding before commenting?

Also this is why you make changelogs to prevent splurge.

**Comment by Dorsisdwarf** [comment]
Why the hell would you have an event that summons narsie, even an admin event. If you want Narsie, spawn narsie. This is a good PR because we can now cultbomb the station without

# **BIG TEXT HAS RISEN**

**Comment by Archie700** [comment]
Also  a admin event of NAR'SIE is pretty much GG ROUND END FUCK YOU GUYS.

No admin who wants to be deadminned immediately would even press this in the middle of a round without very good reasoning and, as Forceful said, Nar'Sie can be spawned from the panel. So, the question is, WHY BOTHER HAVING THE CODE AT ALL IT'S REDUNDANT.

**Comment by Robustin** [comment]
Thank god the sanity squad has arrived. This place will be the end of me.

File: code/modules/events/portal_storm.dm [hunk 3]
```diff
@@ -11,15 +11,15 @@
...
+	name = "Portal Storm: Constructs"
 	typepath = /datum/round_event/portal_storm/portal_storm_narsie
 	weight = 0
 	max_occurrences = 0
 
 /datum/round_event/portal_storm/portal_storm_narsie
-	boss_types = list(/obj/singularity/narsie/large = 1)
-	hostile_types = list(/mob/living/simple_animal/hostile/construct/armored/hostile = 10,\
-						/mob/living/simple_animal/hostile/construct/wraith/hostile = 10)
+	boss_types = list(/mob/living/simple_animal/hostile/construct/builder = 6)
```
Comment (KazeEspada) [inline code/modules/events/portal_storm.dm]: Six artis and no harvestors?

Comment (Archie700) [inline code/modules/events/portal_storm.dm]: Why would you need harvesters

Comment (Robustin) [inline code/modules/events/portal_storm.dm]: Harvesters are supposed to be unique to Nar-Sie, they don't really belong anywhere else.

Comment (xxalpha) [inline code/modules/events/portal_storm.dm]: Because why not? It's a fuck up shit event.

Comment (KorPhaeron) [inline code/modules/events/portal_storm.dm]: The artificers wont spawn until the very end

Comment (xxalpha) [inline code/modules/events/portal_storm.dm]: I think it spreads out the artificers evenly.

Comment (KorPhaeron) [inline code/modules/events/portal_storm.dm]: Thought the boss was at the end of the wave, huh

Comment (ChangelingRain) [inline code/modules/events/portal_storm.dm]: Harvester hostile mobs are boring and weak anyway because they don't cast shit.

**Pull request merged**